pipeline/scripts/ci/reimage-octo-node.sh

```diff
@@ -75,7 +75,7 @@ for node in ${NODES} ; do
     done
 
     echo 'Setting the systems to use shortnames'
-    ssh ${node} 'sudo hostnamectl set-hostname $(hostname -s)'
+    ssh ${node} 'sudo hostnamectl set-hostname $(hostname -s).ceph.redhat.com'
```

Comment:
This may break if other lab systems are consumed like scalelab or beaker. let's make it generic.

Let's improvise the config params. like making `fqdn` as required attr.

Reply:
but question is, what if `hostname -s` is fqdn.